spring-cloud-gateway-server/src/test/java/org/springframework/cloud/gateway/test/sse/SseIntegrationTests.java

```diff
@@ -189,17 +189,17 @@ public void sseAsEventWithoutAcceptHeader() {
 
 		private static final Flux<Long> INTERVAL = interval(Duration.ofMillis(100), 50);
 
-		@RequestMapping("/sse/string")
+		@RequestMapping("/httpbin/sse/string")
```

Comment:
This looks unrelated

Reply:
out unit test define two default filters like bellow:
~~~
      default-filters:
      - AddResponseHeader=X-Response-Default-Foo, Default-Bar
      - PrefixPath=/httpbin
~~~
when one request(url = /sse/string) through PrefixPath filter the request url will be changed to /httpbin/sse/string,if not make this change we will get an 404 error.